app/__init__.py

```diff
@@ -1,12 +1,14 @@
-from flask import Flask
+from flask import Flask, Blueprint
 from flask_restful import Resource, Api
 
 # local import
 from instance.config import app_config
+from .api.v1.views import redflag as rf
```

Comment:
Names should be meaningful to aid in readability and documentation of code
```suggestion
from .api.v1.views import redflag
```